tests/PySys/tedge/tedge_software_update/run.py

```diff
@@ -13,7 +13,7 @@
 TEDGE_DOWNLOAD_DIR = "/tedge_download_dir"
 TEDGE_DOWNLOAD_PATH = "tmp.path"
 TOPIC = 'tedge/commands/req/software/update'
-PAYLOAD = '{"id":"1234","updateList":[{"type":"apt","modules":[{"name":"rolldice","version":"::apt","url":"https://t48415.basic.stage.c8y.io/inventory/binaries/1202","action":"install"}]}]}'
+PAYLOAD = '{"id":"1234","updateList":[{"type":"apt","modules":[{"name":"rolldice","version":"::apt","url":"https://thin-edge-io.eu-latest.cumulocity.com/inventory/binaries/11643549","action":"install"}]}]}'
```

Comment:
Since you are importing the EnvironmentC8y, you can use c8y_url from that instead of hard coding the URL here.

Reply:
Thanks, we still have a hardcoded ID for the binary, we probably should also invent a way to manage them (later).